web/js/tasks.js

```diff
@@ -1097,15 +1097,15 @@ Ext.onReady(function(){
 
 
     // Summary Panel
-    var userWorkSummaryTitleStyle = 'font-weight: bold; width:190px;';
+    var userWorkSummaryTitleStyle = 'font-weight: bold; width:190px; padding: 0px;';
     var summaryPanel = new Ext.FormPanel({
         width: 204,
         labelWidth: 100,
         renderTo: Ext.get('summarypanel'),
         frame:true,
         title: 'User Work Summary',
         defaults: {
-            labelStyle: 'padding: 2px 0 0',
+            labelStyle: 'padding: 1px 0 0',
             style: 'text-align: right;',
```

Comment:
You have to force the same padding rule in the `style` (that's the style applied to the right column), otherwise it defaults to 2px and you aren't saving any space.

Reply:
I'm not following. This style is applying and saving some space. I'm actually tempted to set it to `0px 0 0`